domain/controllerconfig/bootstrap/bootstrap.go

```diff
@@ -68,8 +68,10 @@ func InsertInitialControllerConfig(cfg jujucontroller.Config) internaldatabase.B
 			}
 
 			// Update the attributes.
-			if err := tx.Query(ctx, insertStmt, updateKeyValues).Run(); err != nil {
-				return errors.Trace(err)
+			if len(updateKeyValues) > 0 {
```

Comment:
I'm unsure if I like this, but can we add a log stating there were no controller config values? Could you set it as a warning?

Reply:
I've added a warning. I added this for testing really, since it crashes otherwise when you only specify the controllerUUID. The error the would come be before I added this was from SQLair, it seems better to specifically handle this case than just return that.

I can change it to an actual error and add some fake values to the tests if you think that'd be best.

Reply:
Actually, on second thoughts, its pretty hard to get a logger in there, I might as well just error and change the test.